README.md

```diff
@@ -1,10 +1,55 @@
-# default-SEM-model
+# model_seg_rat_axon-myelin_sem
+---
+
+## Model overview
+![image of segmentation obtained from this model](sem_model_preview.png)
+
 AxonDeepSeg default SEM model and testing image. This model works at a resolution of 0.1 micrometer per pixel and was trained on rat spinal cord data collected via a Scanning Electron Microscope (SEM).
 
+## Dependencies
+- [ivadomed](https://ivadomed.org/) commit: e6554281d07b5afef9e68ce8b02e86b02bd68363
+- [axondeepseg](https://axondeepseg.readthedocs.io/en/latest/) commit: 805868e39eddf620c9f3d60d313cadffb1121bfb
+
+## Segment (ADS)
+To segment an image using this model, use
+```
+axondeepseg -t SEM -i <IMG_PATH> -s <PIXEL_SIZE>
+```
+The `-m` option can be omitted in this case because this is a default built-in model.
+
+## Train and test (ivadomed)
+
+### Clone this repository
+In order to train this model, you will need the json configuration file located in this repo.
+```
+git clone https://github.com/axondeepseg/default-SEM-model
+```
+
+### Get the data
+The SEM dataset used to train this model is hosted on github:
+- https://github.com/axondeepseg/data_axondeepseg_sem
+- commit d7884f9ab6cd8f06cba4885e752988a7706f0ede
+
+```
+git clone https://github.com/axondeepseg/data_axondeepseg_sem
+```
+
+### Train this model
+To train the model, please first update the following fields in the json training config file:
+- `fname_split`: full path to the split_dataset.joblib file
+- `path_data`: full path to training data
+- `gpu_ids`: specific to your hardware
```

Comment:
The order of these items to be changed should be the same order that they are found in the document, in my opinion

Reply:
I see more than one instance of this key, do I need to change all of them or just one specific one?
<img width="261" alt="Screen Shot 2022-03-02 at 12 01 04 AM" src="https://user-images.githubusercontent.com/1421029/156292622-28b71c3d-0d9e-4d87-8de7-20996871f186.png">

Reply:
Same for output and data

Reply:
hm yes the other fields are for object detection, which we don't even use. I added the "parent" key of these hopefully it's more clear now.